cmd/eval.go

```diff
@@ -810,3 +817,45 @@ func generateOptimizedBundle(params evalCommandParams, asBundle bool, filter loa
 
 	return compiler.Bundle(), nil
 }
+
+func enabledStrictFlag(params evalCommandParams) (err error) {
```

Comment:
[nit] there isn't often a real need to name returned values
```suggestion
func enabledStrictFlag(params evalCommandParams) error {
```

Reply:
Removed! :wave: